lib/css/components/_stacks-toggle-switches.less

```diff
@@ -10,76 +10,142 @@
 //  $   BASE STYLE
 //  ----------------------------------------------------------------------------
 .s-toggle-switch {
-    display: flex;
-    align-items: stretch;
-
-    //  First hide the radio buttons
-    input[type="radio"] {
-        position: absolute;
-        left: -999em;
+    input[type="checkbox"] {
+        // "Hide" it by moving it out of sight
+        // This allows users to still "tab" to it, interact via keyboard, etc
         opacity: 0;
-        margin: 0;
-    }
+        position: absolute;
+        left: -99999999px;
+
+        + label {
```

Comment:
I don't really like the idea of <s>Milhouse having two spaghetti meals in one day</s> an input having two labels, one real and the other empty.  We can get into a weird positional thing where if the real label is after the empty one, the description isn't announced.  Then the input would need an `aria-labeledby`.  This is an edge case but one we could avoid with the previous suggestion, we could replace:

```less
+ label {
   ...
}
```

with

```less
+ .s-toggle-switch--indicator {
    pointer-events: none;
}
```

This would let users click directly on the checkbox itself and leave the page with a single label.

```html
<div class="grid ai-center">
    <label class="grid--cell s-label" for="toggle-example-1">…</label>
    <div class="grid--cell s-toggle-switch">
        <input id="toggle-example-1" type="checkbox">
        <div class="s-toggle-swich--indicator"></div>
    </div>
</div>
```